commands/hours.js

```diff
@@ -1,63 +1,90 @@
-const { SlashCommandBuilder } = require('discord.js')
-const { ActionRowBuilder, ButtonBuilder, ButtonStyle } = require('discord.js')
-const { EmbedBuilder } = require('discord.js')
-const fetch = require('node-fetch')
+const { SlashCommandBuilder } = require('discord.js');
+const { ActionRowBuilder, ButtonBuilder, ButtonStyle } = require('discord.js');
+const { EmbedBuilder } = require('discord.js');
+const fetch = require('node-fetch');
+
 module.exports = {
   data: new SlashCommandBuilder()
     .setName('hours')
     .setDescription('See how many hours you have on the network!'),
-  async execute (interaction) {
-    const response = await fetch(`https://callsigns.thepilotclub.org/DiscordOperations/GetVatsimHrsInfo?Discordid=${interaction.user.id}`, {
-      method: 'POST'
-    })
-    const body = await response.json()
-    if (body === 'Not Found') {
+  async execute(interaction) {
+    const response = await fetch(`https://api.vatsim.net/v2/members/discord/${interaction.user.id}`, {
+      method: 'GET',
+    });
+
+    if (!response || response.status !== 200) {
+      console.log(`The Response could not be completed as dialed for ${interaction.member.displayName}`);
+      interaction.reply({ content: 'This Function could not be completed as dialed. Please try again later', ephemeral: true });
+      return;
+    }
+
+    const body = await response.json();
+    const notFound = body.detail;
+    const cid = body.user_id;
+
+    if (notFound === 'Not Found') {
       const row = new ActionRowBuilder()
         .addComponents(
           new ButtonBuilder()
             .setLabel('Connect my account!')
-            .setURL(`https://callsigns.thepilotclub.org/sendauthentication.aspx?id=${interaction.user.id}`)
+            .setURL('https://community.vatsim.net')
             .setStyle(ButtonStyle.Link)
-        )
-      await interaction.reply({ content: 'Please connect your VATSIM account to the TPC Discord!', components: [row], ephemeral: true })
+        );
+      await interaction.reply({
+        content: 'Please connect your VATSIM account to the VATSIM Community Hub!',
+        components: [row],
+        ephemeral: true,
+      }).catch(err => {
+        console.log(err);
+      });
     } else {
-      const data = JSON.parse(body)
+      const hoursResponse = await fetch(`https://api.vatsim.net/v2/members/${cid}/stats`);
+
+      if (!hoursResponse || hoursResponse.status !== 200) {
+        console.log(`The hoursResponse could not be completed as dialed for ${interaction.member.displayName}`);
```

Comment:
We should also include a interaction reply with this so that it does not just error out to the user